src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/Xaml/Parser/SpecialBracketCharacters.cs

```diff
@@ -106,7 +106,7 @@ internal bool EndsEscapeSequence(char ch)
 
         internal bool Match(char start, char end)
         {
-            return _endChars.IndexOf(end.ToString()) == _startChars.IndexOf(start.ToString());
+            return _endChars.IndexOf(end.ToString(), StringComparison.Ordinal) == _startChars.IndexOf(start.ToString(), StringComparison.Ordinal);
```

Comment:
This is the only behavior change to use Ordinal by instead of the default current culture.

It is more inline with the usage of Contains, which defaults to Ordinal comparison here:
https://github.com/dotnet/wpf/blob/78e93de6c2450b4f7b1e3adb5a8b357ab8823d41/src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/Xaml/Parser/SpecialBracketCharacters.cs#L94
and here:
https://github.com/dotnet/wpf/blob/78e93de6c2450b4f7b1e3adb5a8b357ab8823d41/src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/Xaml/Parser/SpecialBracketCharacters.cs#L99